docs/docs/sample-scripts/spo/add-ci-cd-pipeline/index.mdx

```diff
@@ -0,0 +1,292 @@
+---
+tags:
+  - DevOps
+  - pipelines
+  - CI/CD
+  - workflows
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Add CI/CD Pipeline for your SharePoint Framework project
+
+Having a flow that builds and ships your project on push to your repository main branch is a great practice that allows you to share your latest changes with your team, testers or customers. 
+This may save you a lot of time and effort, as you don't need to manually build and deploy your project every time you make a change.
+This script will help you to create a CI/CD pipeline for your SharePoint Framework project. 
+Depending on what parameters you pass, the script will create either a new GitHub Actions workflow or Azure DevOps pipeline.
+If you choose to create application login method (which is default and recommended), the script will also create an app registration in your tenant and generate a self-signed certificate that will be used to authenticate the app.
```

Comment:
```suggestion
Having a flow that builds and ships your project on push to your repository's main branch is a great practice that allows you to share your latest changes with your team, testers or customers. 
This may save you a lot of time and effort, as you don't need to manually build and deploy your project every time you make a change.
This script will help you to create a CI/CD pipeline for your SharePoint Framework project. 
Depending on what parameters you pass, the script will create either a new GitHub Actions workflow or an Azure DevOps pipeline.
If you choose to create an application login method (which is default and recommended), the script will also create an app registration in your tenant and generate a self-signed certificate that will be used to authenticate the app.
```